app/controllers/peeps.rb

```diff
@@ -0,0 +1,36 @@
+class App < Sinatra::Base
+  get '/' do
+    redirect '/peeps'
+  end
+
+  get '/peeps' do
+    @user = current_user(session[:user_id])
+    @peeps = Peep.all(:order => [ :created_at.desc ])
+    erb :'peeps/index'
+  end
+
+  get '/peeps/add' do
+    if @user = current_user(session[:user_id])
```

Comment:
Lint/AssignmentInCondition: Assignment in condition - you probably meant to use `==`. (https://github.com/bbatsov/ruby-style-guide#safe-assignment-in-condition)